app/controllers/shared/api.php

```diff
@@ -243,16 +244,20 @@
                     $user = new Document([
                         '$id' => '',
                         'status' => true,
+                        'type' => Auth::ACTIVITY_TYPE_APP,
                         'email' => 'app.' . $project->getId() . '@service.' . $request->getHostname(),
                         'password' => '',
-                        'name' => $project->getAttribute('name', 'Untitled'),
+                        'name' => 'Dynamic Key',
```

Comment:
It'll be a bit of a change but I wonder if it'll be useful to include the function name here. I'm thinking we'd have to include it in the encoded JWT.

Reply:
what type of function name? call site name?
we could add maybe, that goes out of scope for this one.